unfoldingWord/translationCore
# Cleaned up the home screen implementation

#### This pull request addresses:
- Cleaned up the home screen implementation

#2174
#2254

#### How to test this pull request:
- Test all functionalities in the home screen.
- Click on the (i) on the bottom of the home screen a modal with the licenses should open.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2250)
